printing/ppvernac.ml

```diff
@@ -1199,7 +1199,7 @@ open Decl_kinds
       | VernacSubproof None ->
         return (str "{")
       | VernacSubproof (Some i) ->
-        return (keyword "BeginSubproof" ++ spc () ++ int i)
```

Comment:
I should have seen this `BeginSubproof` but I was just fixing expected typing errors ;)